src/TimeInput.js

```diff
@@ -80,19 +89,21 @@ class TimeInput extends React.Component {
         value={formattedValue}
         readOnly
         key='TimeInput-input'
+        classes={inputClasses || {}}
```

Comment:
No reason to set this to an empty object. Also, instead of introducing a new prop, this could as well be `classes.input` (just add `input: {}` to the styles object above, the user will then be able to customize it with `classes={{ input: 'the-users-input-class' }}`, a common Material-UI pattern).